infra/scripts/docker_build_cross_arm_compiler_debug.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+[[ "${BASH_SOURCE[0]}" != "${0}" ]] && echo "Please don't source ${BASH_SOURCE[0]}, execute it" && return
+
+CURRENT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ROOT_PATH="$CURRENT_PATH/../../"
+
+# prepare rootfs
+if [ -z "$ROOTFS_DIR" ] || [ ! -d $ROOTFS_DIR ]; then
+  echo "It will use default rootfs path"
+else
+  DOCKER_VOLUMES+=" -v $ROOTFS_DIR:/opt/rootfs"
+  DOCKER_ENV_VARS+=" -e ROOTFS_ARM=/opt/rootfs"
+fi
+
+# mount volume (or directory) for externals
+if [ -n "$EXTERNAL_VOLUME" ]; then
+  DOCKER_VOLUMES+=" -v $EXTERNAL_VOLUME:/externals"
+  DOCKER_ENV_VARS+=" -e EXTERNAL_VOLUME=/externals"
+else
+  echo "It will use default external path"
+fi
+
+DOCKER_ENV_VARS+=" -e BUILD_TYPE=Debug"
+DOCKER_ENV_VARS+=" -e BUILD_JOBS=4"
+
+# docker image name
+# - for bionic, use DOCKER_IMAGE_NAME="nnfw/one-devtools:bionic"
```

Comment:
@hseok-oh , is this comment OK? or is there for `nncc` ?

Reply:
It's OK. Compiler and runtime share docker image.